espeak-ng not set to default speech synthesis engine

### Steps to reproduce

Install `espeak-ng`. `spd-conf -u`, set `espeak-ng` as default speech synthesis engine.

### Obtained behavior

`spd-say -)` lists `espeak-ng-mbrola`. I never installed `espeak-ng-mbrola` that I am aware of.

### Expected behavior

`speech-dispatcher` to work as it used to.
### Behavior information

Please follow the next step, to provide us with precious information about how things went wrong on your machine:

* Set `LogLevel` to 5 in your `/etc/speech-dispacher/speechd.conf`
* Restart your computer to make sure it gets taken into account
* Reproduce the issue
* Make a copy of the `speech-dispacher.log` file, it is usually found in `/run/user/1000/speech-dispatcher/log`. Also make a copy of `yourspeechmodule.log` (e.g. `espeak-ng.log`).
* Set `LogLevel` back to 3 in your `/etc/speech-dispacher/speechd.conf`
* Attach the `speech-dispatcher.log` and `yourspeechmodule.log` file to your bug report.

> [Fri Apr  7 21:34:50 2023 : 166111] speechd:  Error: Module reported error in request from speechd (code 3xx): 300-Opening sound device failed. Reason: Cannot open plugin pulse. error: file not found. 
> 300 MODULE ERROR
> .
> [Fri Apr  7 21:34:50 2023 : 166133] speechd: ERROR: Can't initialize audio in output module, see reason above.
> [Fri Apr  7 21:34:50 2023 : 166224] speechd:   Speech Dispatcher Logging to file /run/user/999/speech-dispatcher/log//speech-dispatcher.log
> [Fri Apr  7 21:34:50 2023 : 166263] speechd: Error [speechd.c:672]:No speech output modules were loaded - aborting...

### Distribution

Xubuntu 22.04

### Version of Speech-dispatcher

speech-dispatcher 0.11.1 

I also tried building speech-dispatcher from this repository. Different error, still no voices.

I initially observed this testing `window.speechSynthesis.speak()`.

Is there any reason `./configure --with-espeak-ng` says it can't find `espeak-ng` even though the package is installed?

```
checking for espeak_Synth in -lespeak... no
checking for espeak-ng... no
configure: error: in `/home/xubuntu/Downloads/speech-dispatcher-0.11.4':
configure: error: espeak-ng is not available
See `config.log' for more details
```

--- a/src/server/output.c
+++ b/src/server/output.c
@@ -991,6 +991,8 @@ static int output_module_is_speaking(OutputModule * output)
 				MSG(4, "we sent PAUSE too late, now tell the speak queue");
 				if (!output_pause_queued)
 					module_speak_queue_pause();
+				if (!module_speak_queue_add_end())
+					MSG(3, "Warning: couldn't add end to speak queue");
 			} else {
 				if (!module_speak_queue_add_end())
 					MSG(3, "Warning: couldn't add end to speak queue");

